docs/src/doc/docs/user_guide/cli/usage.md

```diff
@@ -0,0 +1,49 @@
+# Using command line
+
+To run Dokka from the command line, download the [Dokka CLI runner](https://github.com/Kotlin/dokka/releases/download/v1.4.0-rc/dokka-cli-1.4.0-rc.jar).
+To generate documentation, run the following command:
+```
+java -jar dokka-cli.jar <arguments>
+```
+
+## Configuration options
+
+Dokka supports the following command line arguments:
+
+  * `-outputDir` - the output directory where the documentation is generated
+  * `-cacheRoot` - cache directory to enable package-list caching
+  * `-pluginsClasspath` - artifacts with dokka plugins, separated by `;`. At least dokka base and all its dependencies must be added there 
+  * `-offlineMode` - do not resolve package-lists online
+  * `-failOnWarning` - throw an exception instead of a warning
+  * `-globalPackageOptions` - per package options added to all source sets
+  * `-globalLinks` - external documentation links added to all source sets
+  * `-globalSrcLink` - source links added to all source sets
+  * `-sourceSet` - (repeatable) - configuration for a single source set. Following this argument, you can pass other arguments:
+    * `-moduleName` - (required) - module name used as a part of source set ID when declaring dependent source sets
+    * `-moduleDisplayName` - displayed module name
```

Comment:
This does not exist anymore! Check recent master for this, I updated the Readme there already 👍

Reply:
Strange, took that from master 🤔 . Presumably I didn't have the most recent one, will fix

Reply:
Merged a few minutes ago 🙈 
🤦 Sorry

Reply:
Done ✅